teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorProposerOptions.java

```diff
@@ -28,15 +28,15 @@
   private String proposerDefaultFeeRecipient = null;
 
   @Option(
-      names = {"--Xvalidators-proposer-config"},
+      names = {"--validators-proposer-config"},
       paramLabel = "<STRING>",
       description = "remote URL or local file path to load proposer configuration from",
       arity = "0..1",
       hidden = true)
```

Comment:
Need to remove the hidden = true as well.

Reply:
Which apparently we forgot for the ExecutionEngineOptions - could you unhide them too please?